src/test/FuzzerImpl.cpp

```diff
@@ -516,27 +745,44 @@ isBadOverlayFuzzerInput(StellarMessage const& m)
     return m.type() == AUTH || m.type() == ERROR_MSG || m.type() == HELLO;
 }
 
-// Empties "ops" as operations are applied.  If `throwIfTxFails` is `false`,
+// Empties "ops" as operations are applied.  If `isSetupOperation` is `false`,
 // ignores failures and might apply further transactions after failures.
-static void
-attemptToApplyOps(LedgerTxn& ltx, PublicKey const& sourceAccount,
-                  xdr::xvector<Operation>::const_iterator begin,
-                  xdr::xvector<Operation>::const_iterator end, Application& app,
-                  bool const throwIfTxFails = true)
+// If `isSetupOperation` is `true`, throws exceptions on transaction failures,
+// and creates new stored ledger key IDs for any new `LedgerEntry`s created by
+// the applied operations.  (When operations are performed by fuzz testing
+// rather than by setup, the new IDs are created when the operations are
+// unpacked.)
+void
+TransactionFuzzer::attemptToApplyOps(
+    LedgerTxn& ltx, PublicKey const& sourceAccount,
+    xdr::xvector<Operation>::const_iterator begin,
+    xdr::xvector<Operation>::const_iterator end, Application& app,
+    bool const isSetupOperation,
+    bool const startWithPostInitializationSequenceNumber)
 {
+    bool nextTxUsesPostInitializationSequenceNumber =
+        startWithPostInitializationSequenceNumber;
+
     for (auto beginOpsInThisTx = begin; beginOpsInThisTx != end;)
     {
         auto endOpsInThisTx =
             std::distance(beginOpsInThisTx, end) <= MAX_OPS_PER_TX
                 ? end
                 : begin + MAX_OPS_PER_TX;
-        auto txFramePtr =
-            createFuzzTransactionFrame(sourceAccount, beginOpsInThisTx,
-                                       endOpsInThisTx, app.getNetworkID());
+
+        auto txFramePtr = createFuzzTransactionFrame(
```

Comment:
I factored the part that's specific to setup operations out into a separate method.  I've also removed excess trace logging, which has shortened `attemptToApplyOps()` further.

It's still called by `inject()` which doesn't need to loop, but I don't think it's harmful that that feature will only be used by the setup code; it has to be written anyway because of the setup code, and `inject()` still uses the restart-sequence-numbers functionality, as well as the gathering together of `createFuzzTransactionFrame()` and `attemptApplication()` into a single call.